packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas/createBuyQuote.ts

```diff
@@ -0,0 +1,31 @@
+import { put, race, take } from 'typed-redux-saga'
```

Comment:
Is the thinking here that we should start abstracting out individual saga functions into their own files? I would be all for that.

Reply:
Yes. To be clear on this, we shall try to have one file per saga used in saga register file as it will make test files more readable. However, inside those files we should be splitting saga as much as it makes sense from readability perspective.